lib/services/constants.dart

```diff
@@ -0,0 +1,18 @@
+import 'package:flutter/widgets.dart';
+import 'package:google_maps_flutter/google_maps_flutter.dart';
+
+const Color COLOR_CONCORDIA = Color.fromRGBO(147, 35, 57, 1);
```

Comment:
Good practice on putting everything in a constants file, this is how it should be done from now on.